### PR TITLE
Allow storing of many rows of followup comments in Vasu documents

### PR DESCRIPTION
**Pull request by zch**
#### Summary
After a Vasu document has been finalized, a followup question is added to what currently is question number 5.

The followup section renders as a text area and when a comment is added, the comment is rendered as a "log row" above the text area. Each "log row" is tagged with the date when it was created as well as the name of the author.

### CODE REVIEW

File: frontend/src/employee-frontend/components/vasu/components/FollowupQuestion.tsx [hunk 1]
```diff
@@ -2,20 +2,88 @@
...
+  onChange
+}: {
+  entry?: FollowupEntry
+  onChange: (entry: FollowupEntry) => void
+}) {
+  const { user } = useContext(UserContext)
+  const { i18n } = useTranslation()
+  const [textValue, setTextValue] = useState(entry ? entry.text : '')
+
+  const onSubmit = () => {
```
Comment (akheron) [inline frontend/src/employee-frontend/components/vasu/components/FollowupQuestion.tsx:56]: Maybe add `useCallback`?

File: frontend/src/employee-frontend/components/vasu/components/FollowupQuestion.tsx [hunk 2]
```diff
@@ -24,11 +92,8 @@ export default React.memo(function FollowupQuestion({
   question: { title, name, value, info },
   translations
 }: FollowupQuestionProps) {
-  const getEditorOrStaticText = () => {
-    if (!onChange) {
-      return <ValueOrNoRecord text={value} translations={translations} />
-    }
-    return <TextArea value={value} onChange={onChange} />
+  const addNewEntry = (entry: FollowupEntry) => {
```
Comment (akheron) [inline frontend/src/employee-frontend/components/vasu/components/FollowupQuestion.tsx:100]: Same here

File: frontend/src/e2e-playwright/pages/employee/vasu/vasu.ts [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+import { Page } from 'playwright'
+import { RawElement } from '../../../utils/element'
+
+export default class VasuPage {
+  constructor(private readonly page: Page) {}
+
+  readonly #followupQuestion = this.page.locator(
+    '[data-qa="vasu-followup-question"]'
+  )
+  readonly #finalizeButton = new RawElement(
```
Comment (akheron) [inline frontend/src/e2e-playwright/pages/employee/vasu/vasu.ts:14]: Use locators instead of `RawElement` in new code

**Review by akheron — CHANGES_REQUESTED** [review]

File: frontend/src/e2e-playwright/pages/employee/vasu/vasu.ts (hunk 3 above)
Comment (zch) [reply frontend/src/e2e-playwright/pages/employee/vasu/vasu.ts]: I went ahead and added a deprecated flag on `RawElement` to avoid this in the future.

**Review by akheron — APPROVED** [review]

**Pull request merged**